### ISSUE TITLE
Registry: Error creating Certificate

### ISSUE THREAD

**Issue by marvinwankersteen** [rancher/community-catalog#120]
Hey,

I have tried to deploy the registry but the sslproxy wont start. I looked a bit around and found the problem with the certificates. There could not created from portus.

The error is:

```
INFO[0066] [2/5] [portus]: Started                      
INFO[0066] [2/5] [registry]: Starting                   
INFO[0066] [2/5] [sslproxy]: Starting                   
portus_1 | 2016-04-26T18:03:10.080710472Z Creating Certificate
2016-04-26T18:03:10.111949368Z error on line 353 of /dev/fd/63
portus_1 | 2016-04-26T18:03:10.111974046Z 140235343283856:error:0E079065:configuration file routines:DEF_LOAD_BIO:missing equal sign:conf_def.c:362:line 353
portus_1 | 2016-04-26T18:03:10.112663656Z config FQDN into rails
```

When I try to execute the "openssl"-Command from the startup.sh I get the following error:

```
#  openssl req -x509 -newkey rsa:2048 -keyout "$PORTUS_KEY_PATH" -out "$PORTUS_CRT_PATH" -days 3650 -nodes -subj "/CN=$PORTUS_MACHINE_FQDN" -extensions SAN -config <(cat /etc/ssl/openssl.cnf <(printf "[SAN]\nsubjectAltName=DNS:registry,DNS:$PORTUS_MACHINE_FQDN,DNS:$ALTNAME,IP:$IPADDR,DNS:portus"))
Error Loading extension section SAN
140057797936784:error:2206D06D:X509 V3 routines:X509V3_parse_list:invalid null value:v3_utl.c:299:
140057797936784:error:22097069:X509 V3 routines:DO_EXT_NCONF:invalid extension string:v3_conf.c:139:name=subjectAltName,section=DNS:registry,DNS:registry.<domain>.net,DNS:,IP:,DNS:portus
140057797936784:error:22098080:X509 V3 routines:X509V3_EXT_nconf:error in extension:v3_conf.c:93:name=subjectAltName, value=DNS:registry,DNS:registry.<domain>.net,DNS:,IP:,DNS:portus
```

I have tried to deploy the containers on two hosts. But I get on both hosts the same error.
I have no idea what exactly is the problem. I hope somone can help me.

**Comment by marvinwankersteen** [comment]
I have found the problem. Its the problem with the IP from this issue: https://github.com/rancher/community-catalog/issues/112

When I add the "|tail -1" then the certs are created.

**Comment by sshipway** [comment]
This is now fixed in community template version `2.3.1-3.1` which uses container version `sshipway/portus:2.0.5`, which has the extra `|tail -1` in place to avoid this situation.
This ticket can be closed.
